python/testing.rst

```diff
@@ -340,14 +361,21 @@ which ends up running the single specified test plus the two running as part of
 
 Note that `~lsst.utils.tests.MemoryTestCase` must always be the
 final test suite.
-For the memory test to function properly the `lsst.utils.tests.init` function must be invoked before any of the tests in the class are executed.
-Since LSST test scripts are required to run properly from the command-line and when called from within `pytest`_, the `~lsst.utils.tests.init` function has to be in the file twice: once in the :ref:`setup_module <pytest:xunitsetup>` function that is called by `pytest`_ whenever a test module is loaded (`pytest`_ will not use the ``__main__`` code path), and also just before the call to `unittest.main` call to handle being called with :command:`python`.
+For the file descriptor test to function properly the `lsst.utils.tests.init` function must be invoked before any of the tests in the class are executed.
+Since LSST test scripts are required to run properly when called from within `pytest`_, the `~lsst.utils.tests.init` function has to be in the :ref:`setup_module <pytest:xunitsetup>` function that is called by `pytest`_ whenever a test module is loaded.
+It is no longer required that this function also be present just before the call to `unittest.main` to handle being called with :command:`python`.
```

Comment:
Although it is useful to leave it in *if* the test file is still solely using unittest.